### PR TITLE
fix #116456: add restart button to mscore

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
and bring up a dialog using it when language got changed or current language's translation got updated

### CODE REVIEW

File: mscore/musescore.cpp [hunk 1]
```diff
@@ -1288,6 +1289,29 @@ void MuseScore::resetAndRestart()
       }
 
 //---------------------------------------------------------
+//   restartAfterSettingsChange
+//---------------------------------------------------------
+
+void MuseScore::restartAfterSettingsChange()
+      {
+      int ret = QMessageBox::question(0, tr("Are you sure?"),
+                  tr("You have changed settings that requite a restart of MuseScore in order to take effect\n"
```
Comment (lasconic) [inline mscore/musescore.cpp]: requite --> require

File: mscore/musescore.cpp [hunk 2]
```diff
@@ -1288,6 +1289,29 @@ void MuseScore::resetAndRestart()
...
+                   "Do you you want to restart MuseScore now?"),
```
Comment (lasconic) [inline mscore/musescore.cpp]: you you --> you

**Comment by Jojo-Schmitz** [comment]
probably largly made obsolete once #2708 gets merged

**Comment by lasconic** [comment]
I merged the dynamic translation PR  #2708 What do we do with that?

**Comment by Jojo-Schmitz** [comment]
I think we can close it.

**Pull request closed without merging**